### DISCUSSION TITLE
Cannot remove domain?

### DISCUSSION THREAD

**Discussion by ZA-OI-DE** [towfiqi/serpbear#238] [General]
I made a mistake when adding a domain, so there is a space and a comma, now it does not show me the edit button and I cannot remove it, can it be done manually?

**Comment by towfiqi** [comment]
try opening the database.sqlite file using sqlite viewer app and removing the domain from the database.

**Comment by syamsulalam** [comment]
you can remove domain by clicking on said domain then click on domain settings, there you'll find remove domain

![image](https://github.com/user-attachments/assets/1f6b5ead-5859-4033-bb45-df0f7fe63898)
